### PR TITLE
🎨 Synced retention policy and minor template changes

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Various minor enhancements after manual exploratory testing

####  Account Deletion
- 🎨  new settings  `LOGIN_ACCOUNT_DELETION_RETENTION_DAYS` (currently in the login but probably all registration will be moved out to its own plugin later on)
- 🎨 **synced** value between front-end and backend via statics data:
image
image

####  Email Templates
- 🐛 Fixed name used in registration email by `first_name`

## How to test

driving test 
```
cd services/web/server
make install-dev
pytest -v -k test_settings_to_client_statics_plugins tests/unit/isolated
```

## Dev-ops checklist

- new `LOGIN_ACCOUNT_DELETION_RETENTION_DAYS`
- [x] osparc-config [MR counterpart](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/481)

### CODE REVIEW

File: .env-devel [hunk 1]
```diff
@@ -235,6 +235,7 @@ WEBSERVER_GARBAGE_COLLECTOR=null
 WEBSERVER_DB_LISTENER=0
 
 WEBSERVER_LOGIN={}
+LOGIN_ACCOUNT_DELETION_RETENTION_DAYS=31
```
Comment (sanderegg) [inline .env-devel:238]: you probably need a MR in osparc-configs

**Review by sanderegg — APPROVED** [review]

File: .env-devel (hunk 1 above)
Comment (pcrespov) [reply .env-devel:238]: The link to that MR is already in the `devops` section of this PR :-)

File: services/static-webserver/client/source/class/osparc/desktop/credits/DeleteAccount.js [hunk 2]
```diff
@@ -39,10 +39,11 @@ qx.Class.define("osparc.desktop.credits.DeleteAccount", {
       switch (id) {
         case "intro-text": {
           const supportEmail = osparc.store.VendorInfo.getInstance().getSupportEmail();
+          const retentionDays = osparc.store.StaticInfo.getInstance().getAccountDeletionRetentionDays();
           const text = this.tr(`\
-            This account will be deleted in 14 days.<br>\
+            This account will be <strong>deleted in ${retentionDays} days</strong>.<br>\
             During this period, if you want to recover it or delete your\
-            data right away, please send us an email to ${supportEmail}.\
+            data right away, please send us an email to <a href="mailto:${supportEmail}">${supportEmail}</a>.\
```
Comment (odeimaiz) [inline services/static-webserver/client/source/class/osparc/desktop/credits/DeleteAccount.js:46]: 👌

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by odeimaiz — APPROVED** [review]
Beautiful 💅

**Review by YuryHrytsuk — APPROVED** [review]

**Pull request merged**